### PR TITLE
[updated] Adding import auth documentation

### PR DESCRIPTION
**Pull request by pananapread**
_Issue #, if available:_
n/a

_Description of changes:_
Adding import authentication documentation for Admin UI's upcoming feature to reuse existing cognito user pools and identity pools

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by pananapread** [comment]
Feature launches July 6th

**Review by siegerts — APPROVED** [review]

File: docs/console/auth/import.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Use an existing Amazon Cognito user pool and identity pool
+description: Configure the Admin UI to use existing Amazon Cognito user pool and identity pool resources as an authentication and authorization mechanism for other Amplify categories ( For example, API, Storage, and more).
+---
+
+You can import existing Amazon Cognito resources into your Amplify project using the Admin UI. To get started, choose **Authentication** from the **Set up** menu in your app's Admin UI. On the **Authentication** page, chose **Reuse existing Amazon Cognito resources**.
```
Comment (yppartyk) [inline docs/console/auth/import.md:6]: `chose` --> `choose`

File: docs/console/auth/import.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Use an existing Amazon Cognito user pool and identity pool
```
Comment (swaminator) [inline docs/console/auth/import.md:2]: Very long title.. it'll show up weirdly in the sidenav. Suggestion 'Import Amazon Cognito resources'

File: docs/console/auth/import.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+description: Configure the Admin UI to use existing Amazon Cognito user pool and identity pool resources as an authentication and authorization mechanism for other Amplify categories ( For example, API, Storage, and more).
+---
+
+You can import existing Amazon Cognito resources into your Amplify project using the Admin UI. To get started, choose **Authentication** from the **Set up** menu in your app's Admin UI. On the **Authentication** page, chose **Reuse existing Amazon Cognito resources**.
+
+This feature is particularly useful if you’re trying to do the following:
```
Comment (swaminator) [inline docs/console/auth/import.md:8]: Align this content with your what's new post bullets

**Review by yppartyk — APPROVED** [review]
LGTM

File: docs/console/auth/import.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+amplify codegen add
+```
+
+After running this command, the following occurs:
+- Your Amplify Library configuration files (aws-exports.js, amplifyconfiguration.json) are automatically populated with your chosen Amazon Cognito resource information
+- Your designated existing Amazon Cognito resource is provided as the authentication and authorization mechanism for all auth-dependent categories (API, Storage and more)
+- Lambda functions are enabled to access the chosen Amazon Cognito resource if you permit it.
+
+## Multi-environment support
+When you create a new environment, Amplify assumes, by default, that you’re managing your app’s Amazon Cognito resources outside of an Amplify project. You’ll be asked to either import a different Amazon Cognito resource or maintain the same Amazon Cognito resource for your app’s auth category.
```
Comment (swaminator) [inline docs/console/auth/import.md:52]: Too many commas here: When you create a new environment, you’ll be asked to either import a different Amazon Cognito resource or maintain the same Amazon Cognito resource for your app’s auth category.

Also functionality-wise, what happens when I clone an env in the Amplify console?

File: docs/console/auth/import.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+If you want to have Amplify manage your authorization resources in a new environment, unlink the imported Cognito resource and add authorization to your new environment. This will create new Amplify-managed authorization resources in the new environment.
+  
+## Unlink an existing Amazon Cognito user pool or identity pool
+To order to unlink your existing Amazon Cognito resource, click **Unlink Cognito User Pool and Identity Pool** on the bottom of the **Authentication** page and follow the prompt to confirm this action. This only unlinks the Amazon Cognito resource referenced from the Amplify project. It does not delete the Amazon Cognito resource itself.
```
Comment (swaminator) [inline docs/console/auth/import.md:57]: Typo: In order to..

**Review by swaminator — CHANGES_REQUESTED** [review]

File: docs/console/auth/import.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+amplify codegen add
+```
+
+After running this command, the following occurs:
+- Your Amplify Library configuration files (aws-exports.js, amplifyconfiguration.json) are automatically populated with your chosen Amazon Cognito resource information
+- Your designated existing Amazon Cognito resource is provided as the authentication and authorization mechanism for all auth-dependent categories (API, Storage and more)
+- Lambda functions are enabled to access the chosen Amazon Cognito resource if you permit it.
+
+## Multi-environment support
+When you clone an environment or create a new one, you’ll be asked to either import a different Amazon Cognito resource or maintain the same Amazon Cognito resource for your app’s auth category.
```
Comment (swaminator) [inline docs/console/auth/import.md:52]: Update to match existing experience when cloning a backend env in the console

File: docs/console/auth/import.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+- Manage users and groups from the Admin UI (instead of having to login to the AWS console)
+- Reuse the imported auth resource across Amplify environments
+
+## Import an existing Cognito user pool
+To successfully import your user pool, your user pools require at least two app clients with the following conditions:
+
+- At least one “Web app client”: an app client without a client secret
+- At least one “Native app client“: an app client with a client secret
+To complete the import procedure, run the `amplify push` command.
+
```
Comment (swaminator) [inline docs/console/auth/import.md:20]: Add a blurb on why this is needed.

**Review by swaminator — CHANGES_REQUESTED** [review]

**Review by swaminator — APPROVED** [review]

**Pull request merged**